service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -16,9 +16,21 @@
 
 import java.util.Arrays;
 
+/**
+ * Lagom service implementations must create one implementation of this interface and use it to bind Service
+ * implementations.
+ */
 public interface ServiceGuiceSupport extends ServiceClientGuiceSupport {
 
-    // redefines bindServiceInfo from super interface.
+    /**
+     * Creates a custom {@link ServiceInfo} for this Lagom service. This method overrides
+     * {@link ServiceClientGuiceSupport#bindServiceInfo(ServiceInfo)} with custom behavior for consume-only Lagom
+     * services.
+     * <p>
+     * The customization may be used by Lagom to add features only applicatble to Lagom services transparently.
```

Comment:
I don't think that what we are deprecating _multiple services_, what we are deprecating is multiple _locatable_ services. I think the need to bundle many service descriptors into a single service will still exist: adding metrics support, or swagger file delivery, or monitor/tooling/etc... is still useful.